frontend/src/app/update-file/update-file.component.ts

```diff
@@ -51,6 +68,27 @@ export class UpdateFileComponent implements OnChanges {
       this.selectedFiles = [currentValue];
     }
   }
+  readURL(event:any) {
```

Comment:
issue: replace `any` with discrete type.

Reply:
fixed in the latest commit https://github.com/GoogleCloudPlatform/app-large-data-sharing-golang/pull/1/commits/6b656573d3db05f371011f3a0ae3f416e9b4fc74